src/Microsoft.Graph.Auth/ConfidentialClient/OnBehalfOfProvider.cs

```diff
@@ -41,18 +43,28 @@ public class OnBehalfOfProvider : MsalAuthenticationBase, IAuthenticationProvide
         /// <param name="clientId">Client ID (also known as <i>Application ID</i>) of the application as registered in the application registration portal (https://aka.ms/msal-net-register-app).</param>
         /// <param name="redirectUri">also named <i>Reply URI</i>, the redirect URI is the URI where the STS (Security Token Service) will call back the application
         ///  with the security token. For details see https://aka.ms/msal-net-client-applications.</param>
-        /// <param name="clientCredential">A <see cref="Microsoft.Identity.Client.ClientCredential"/> created either from an application secret or a certificate.</param>
-        /// <param name="tokenStorageProvider">A <see cref="ITokenStorageProvider"/> for storing and retrieving access token.</param>
-        /// <param name="tenant">Tenant to sign-in users. This defaults to <c>common</c> if non is specified.</param>
-        /// <param name="nationalCloud">A <see cref="NationalCloud"/> which identifies the national cloud endpoint to use as the authority. This defaults to the global cloud <see cref="NationalCloud.Global"/> (https://login.microsoftonline.com).</param>
+        /// <param name="clientSecret">A client secret.</param>
+        /// <param name="tenant">Tenant to sign-in users. This defaults to <see cref="AadAuthorityAudience.AzureAdMultipleOrgs" /> if none is specified.</param>
+        /// <param name="cloud">A <see cref="AzureCloudInstance"/> which identifies the cloud endpoint to use as the authority. This defaults to the public cloud <see cref="AzureCloudInstance.AzurePublic"/> (https://login.microsoftonline.com).</param>
         /// <returns>A <see cref="IConfidentialClientApplication"/></returns>
         /// <exception cref="AuthenticationException"/>
-        public static IConfidentialClientApplication CreateClientApplication(string clientId,
+        public static IConfidentialClientApplication CreateClientApplication(
```

Comment:
We can get rid of this static create method.

Reply:
Looks like I missed that one. Done!